DOCUMENTATION.md

```diff
@@ -3927,11 +3931,178 @@ const cfg = {
 The client calls the `init()` method with the `properties` configuration option. Then the client calls the `getSSLOptions()`
 method of `SSLOptionsFactory` to create the `options` object.
 
-## 9.2. Credentials
+## 9.2 Authentication
 
-One of the key elements in Hazelcast security is the `Credentials` object, which can be used to carry all security attributes of
-the Hazelcast Node.js client to Hazelcast members. Then, Hazelcast members can authenticate the clients and perform access
-control checks on the client operations using this `Credentials` object.
+By default, the client does not use any authentication method and just uses the configured cluster name to connect to members.
```

Comment:
We may also add a warning about enabling SSL, otherwise the username/password/token may be easily eavesdropped by third parties. Do you think we should also mention that authentication is an enterprise feature?

Reply:
I had a warning about the SSL at the end, but I have moved it to top. Also, put a note about it requiring enterprise edition